lib/cmd/fh3/credential/download.js

```diff
@@ -0,0 +1,40 @@
+/* globals i18n */
+var fhreq = require('../../../utils/request.js');
+var fs = require('fs');
+var util = require('util');
+
+module.exports = {
+  'desc' : i18n._('Download credential bundle'),
+  'examples' : [{
+    cmd : 'fhc credential download --id=<bundle-id> --output=<Path To Output File>',
```

Comment:
This already works as following the local test. I changed the text in the description for the example be "(E.g --privateKey:/Users/user/)"

```
Camilas-MBP:fh-fhc cmacedo$ ./bin/fhc.js credential download --id=ysr5sttwy7bualszdhwtwwiy --output=/Users/cmacedo/
(node) sys is deprecated. Use util instead.
Saved to /Users/cmacedo/ysr5sttwy7bualszdhwtwwiy.zip

Camilas-MBP:fh-fhc cmacedo$ 
```